docs/docs/cmd/spo/file/file-roleassignment-remove.mdx

```diff
@@ -23,13 +23,19 @@ m365 spo file roleassignment remove [options]
 : The UniqueId (GUID) of the file. Specify either `fileUrl` or `fileId` but not both.
 
 `--principalId [principalId]`
-: The SharePoint Id of the principal. It may be either a user id or group id. Specify either `upn`, `groupName`, or `principalId` but not multiple.
+: The SharePoint Id of the principal. It may be either a user id or group id. Specify either `upn`, `groupName`, entraGroupId, entraGroupName, or `principalId` but not multiple.
```

Comment:
```suggestion
: The SharePoint Id of the principal. It may be either a user id or group id. Specify either `upn`, `groupName`, `entraGroupId`, `entraGroupName`, or `principalId` but not multiple.
```